dev/create-release/spark-rm/Dockerfile

```diff
@@ -42,8 +42,8 @@ ARG PIP_PKGS="pyopenssl pypandoc numpy pygments sphinx"
 #
 # This is all in a single "RUN" command so that if anything changes, "apt update" is run to fetch
 # the most current package versions (instead of potentially using old versions cached by docker).
-RUN apt-get clean && apt-get update && $APT_INSTALL gnupg && \
-  echo 'deb http://cran.cnr.Berkeley.edu/bin/linux/ubuntu bionic-cran35/' >> /etc/apt/sources.list && \
+RUN apt-get clean && apt-get update && $APT_INSTALL gnupg && $APT_INSTALL ca-certificates \
```

Comment:
@srowen . This fails because `&&` is missing at the end of the line. 
```
E: Unable to locate package echo
E: Unable to locate package deb [trusted=yes] https://cloud.r-project.org/bin/linux/ubuntu bionic-cran35
E: Couldn't find any package by glob 'deb [trusted=yes] https://cloud.r-project.org/bin/linux/ubuntu bionic-cran35'
E: Couldn't find any package by regex 'deb [trusted=yes] https://cloud.r-project.org/bin/linux/ubuntu bionic-cran35'
The command '/bin/sh -c apt-get clean && apt-get update && $APT_INSTALL gnupg && $APT_INSTALL ca-certificates   echo 'deb [trusted=yes] https://cloud.r-project.org/bin/linux/ubuntu bionic-cran35/' >> /etc/apt/sources.list &&   gpg --keyserver keyserver.ubuntu.com --recv-key E084DAB9 &&   gpg -a --export E084DAB9 | apt-key add - &&   apt-get clean &&   rm -rf /var/lib/apt/lists/* &&   apt-get clean &&   apt-get update &&   $APT_INSTALL software-properties-common &&   apt-add-repository -y ppa:brightbox/ruby-ng &&   apt-get update &&   $APT_INSTALL openjdk-8-jdk &&   update-alternatives --set java /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java &&   $APT_INSTALL curl wget git maven ivy subversion make gcc lsof libffi-dev     pandoc pandoc-citeproc libssl-dev libcurl4-openssl-dev libxml2-dev &&   curl -sL https://deb.nodesource.com/setup_11.x | bash &&   $APT_INSTALL nodejs &&   $APT_INSTALL libpython2.7-dev libpython3-dev python-pip python3-pip &&   pip install $BASE_PIP_PKGS &&   pip install $PIP_PKGS &&   cd &&   virtualenv -p python3 /opt/p35 &&   . /opt/p35/bin/activate &&   pip install $BASE_PIP_PKGS &&   pip install $PIP_PKGS &&   $APT_INSTALL r-base r-base-dev &&   $APT_INSTALL texlive-latex-base texlive texlive-fonts-extra texinfo qpdf &&   Rscript -e "install.packages(c('curl', 'xml2', 'httr', 'devtools', 'testthat', 'knitr', 'rmarkdown', 'roxygen2', 'e1071', 'survival'), repos='https://cloud.r-project.org/')" &&   Rscript -e "devtools::install_github('jimhester/lintr')" &&   $APT_INSTALL ruby2.3 ruby2.3-dev mkdocs &&   gem install jekyll --no-rdoc --no-ri &&   gem install jekyll-redirect-from &&   gem install pygments.rb' returned a non-zero code: 100
```

After fixing like `$APT_INSTALL ca-certificates \` -> `$APT_INSTALL ca-certificates && \`, you can test with the following command.
```
$ cd dev/create-release
$ docker build -t "test" spark-rm
```

Reply:
Arg, sorry, try my latest commit here. It seems like we don't need `[trusted=yes]`, just the certificates

Reply:
Thank you~